scripts/v.dissolve/v.dissolve.html

```diff
@@ -1,126 +1,176 @@
+<b>aggregate_columns</b> can be used to specify the method using SQL 
+syntax. This provides the highest flexibility, and it is suitable for  
+scripting. The SQL statement should specify both the column and the 
+functions applied, e.g.,
 
 <p>
-Aggregate methods are specified by name in <b>aggregate_methods</b>
-or using SQL syntax in <b>aggregate_columns</b>.
-If <b>result_columns</b> is provided including type information
-and the <em>sql</em> backend is used,
-<b>aggregate_columns</b> can contain SQL syntax specifying both columns
-and the functions applied, e.g.,
 <code>aggregate_columns="sum(cows) / sum(animals)"</code>.
-In this case, <b>aggregate_methods</b> should to be omitted.
-This provides the highest flexibility and it is suitable for scripting.
 
 <p>
-The backend is, by default, determined automatically based on the requested
-methods. Specifically, the <em>sql</em> backend is used by default,
-but when a method is not one of the SQLite build-in aggregate functions
-and, at the same time, is available with the <em>univar</em> backend,
-the <em>univar</em> backed is used.
-The default behavior is intended for interactive use and testing.
-For scripting and other automated usage, specifying the backend explicitly
-is strongly recommended.
+Note that when the <b>aggregate_columns</b> parameter is used, the 
+<i>sql</i> backend should be used. In addition, the 
+<b>aggregate_columns</b> and <b>aggregate_methods</b> cannot be used 
+together.
 
 <p>
-For convenience, certain methods, namely <em>n</em>, <em>count</em>,
-<em>mean</em>, and <em>avg</em>, are converted to the appropriate name 
-for the selected backend. However, for scripting, it is recommended to
-specify the appropriate method (function) name for the backend, as the 
-conversion is a heuristic that may change in the future.
+For convenience, certain methods, namely <em>n</em>, <em>count</em>, 
+<em>mean</em>, and <em>avg</em>, are automatically converted to the 
+appropriate name for the selected backend. However, for scripting, it 
+is recommended to specify the appropriate method (function) name for 
+the backend, as the conversion is a heuristic that may change in the 
+future.
 
 <p>
-If only <b>aggregate_columns</b> is provided, methods default to
-<em>n</em>, <em>min</em>, <em>max</em>, <em>mean</em>, and <em>sum</em>.
-If the <em>univar</em> backend is specified, all the available methods
-for the <em>univar</em> backend are used.
+If the <b>result_columns</b> is not provided, each method is applied to 
+each column specified by <b>aggregate_columns</b>. This results in 
+columns for all combinations. These result columns have auto-generated 
+names based on the aggregate column and method. For example, setting 
+the following parameters:
+
+<div class="code"><pre>
+aggregate_columns=A,B
+aggregate_methods=sum,n
+</pre></div>
+
+<p>
+Results in the following columns: A_sum, A_n, B_sum, B_n. Also see 
```

Comment:
Is this common to start here with a capital letter when it is continuation of a sentence from above? You could also split the "For example..." part and the result part into two sentences to avoid this completely.

Same issue couple lines above.